src/components/NcDateTimePickerNative/NcDateTimePickerNative.vue

```diff
@@ -159,15 +159,15 @@ export default defineComponent({
 		 * default type: null
 		 */
 		min: {
-			type: [Date, Boolean],
+			type: [Date, Boolean, null],
```

Comment:
We really should get rid of all that `null` values where instead `undefined` is meant